src/styles/main.scss

```diff
@@ -0,0 +1,157 @@
+@import 'colors';
+@import 'measures';
+
+.container {
+  padding: $paddingV-card $paddingH-card;
+  display: grid;
+  grid-template-columns: repeat(4, $width-card);
+  gap: $row-gap $column-gap;
+  justify-content: center;
+  box-sizing: border-box;
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $width-card);
+  }
+
+  @media (max-width: 768px) {
+    grid-template-columns: repeat(2, $width-card);
+  }
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, $width-card);
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  width: $width-card;
+  height: 408px;
+  border-radius: 5px;
+  border: 1px solid $color-border-card;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 32px 19px 36px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    width: 166px;
+    height: 36px;
```

Comment:
Be mindful of setting both width and height on elements that contain text. This can lead to non-responsive behavior or text overflow. Consider using `max-width` or `min-width` to provide better flexibility.